### PR TITLE
[Docs] Baseline actor docs and move around some other files as well

### PR DESCRIPTION
**Pull request by Olshansk**
## Summary

A few small updates to documentation:

- Put a stake in the ground on `Actor` documentation (MVP)
- Lleave some placeholders of the protocol documentation to come.
- Move around package related documentation
- Make configs (which are very detailed) more visible by linking to it from the actor docs

This is intende

## Issue

- #306 

## Type of change

Select one or more:

- [ ] New feature, functionality or library
- [ ] Bug fix
- [ ] Code health or cleanup
- [x] Documentation
- [ ] Other (specify)

## Testing

- [ ] **Run all unit tests**: `make go_develop_and_test`
- [ ] **Run E2E tests locally**: `make test_e2e`
- [ ] **Run E2E tests on DevNet**: Add the `devnet-test-e2e` label to the PR. This is VERY expensive, only do it after all the reviews are complete.

## Sanity Checklist

- [ ] I have tested my changes using the available tooling
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, updated documentation and left TODOs throughout the codebase

### CODE REVIEW

File: docusaurus/docs/actors/application.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- [CLI](#cli)
+
+## Overview
+
+An `Application` is responsible for staking POKT in order to consume and pay for
+services available on Pocket Network as a function of volume and time.
+
+## Schema
+
+The on-chain for an `Application` can be found at [application.proto](./../../../proto/pocket/application/application.proto).
```
Comment (red-0ne) [inline docusaurus/docs/actors/application.md:20]:
```suggestion
The on-chain representation of an `Application` can be found at [application.proto](./../../../proto/pocket/application/application.proto).
```

This makes me ask myself: "on-chain what?"

File: docusaurus/docs/actors/gateway.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- [CLI](#cli)
+
+## Overview
+
+An `Gateway` is responsible for staking POKT in order to relay and sign
+requests on behalf of an `Application`.
+
+## Schema
+
+The on-chain for an `Gateway` can be found at [gateway.proto](./../../../proto/pocket/gateway/gateway.proto).
```
Comment (red-0ne) [inline docusaurus/docs/actors/gateway.md:20]: Same, "on-chain what?"

Comment (red-0ne) [inline docusaurus/docs/actors/gateway.md:20]:
```suggestion
The on-chain for a `Gateway` can be found at [gateway.proto](./../../../proto/pocket/gateway/gateway.proto).
```

File: docusaurus/docs/actors/relay_miner.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+# RelayMiner <!-- omit in toc -->
+
+- [Overview](#overview)
+- [Configuration](#configuration)
+- [CLI](#cli)
+
+## Overview
+
+An `RelayMiner` is responsible for for relaying requests between a client dApp
```
Comment (red-0ne) [inline docusaurus/docs/actors/relay_miner.md:14]: I think you wanted to say: Relaying requests between the `AppGate Server` and the `Service`

File: docusaurus/docs/actors/relay_miner.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+(e.g. mobile app, web app, etc...) and the `Supplier`s on Pocket Network, handling
+all intermediary business logic.
+
+A `Gateway` operator or a sovereign `Application` interested in accessing Pocket
```
Comment (red-0ne) [inline docusaurus/docs/actors/relay_miner.md:18]: This paragraph should talk about the `Supplier` interested in proposing `Services`

File: docusaurus/docs/actors/relay_miner.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+Network directly would need to run an `AppGate Server` or custom software that
+implements the same functionality.
+
+## Configuration
+
+Configurations to stake an `AppGate Server` can be found [relayminer_config.md](../configs/relayminer_config.md).
```
Comment (red-0ne) [inline docusaurus/docs/actors/relay_miner.md:24]: This should talk about configuration for operating a `RelayMiner`.

File: docusaurus/docs/actors/relay_miner.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+## CLI
+
+All of the operations needed to start and operate an `AppGate Server` can be viewed by running:
```
Comment (red-0ne) [inline docusaurus/docs/actors/relay_miner.md:28]:
```suggestion
All of the operations needed to start and operate a `RelayMiner` can be viewed by running:
```

File: docusaurus/docs/actors/appgate_server.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+## Overview
+
+An `AppGate Server` is responsible for relaying requests between a client dApp
+(e.g. mobile app, web app, etc...) and the `Supplier`s on Pocket Network, handling
+all intermediary business logic.
+
+A `Gateway` operator or a sovereign `Application` interested in accessing Pocket
+Network directly would need to run an `AppGate Server` or custom software that
+implements the same functionality.
```
Comment (red-0ne) [inline docusaurus/docs/actors/appgate_server.md:20]: Add a link referring to `POKTRollSDK`

File: docusaurus/docs/actors/appgate_server.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+- [Configuration](#configuration)
+- [CLI](#cli)
+
+## Overview
+
+An `AppGate Server` is responsible for relaying requests between a client dApp
+(e.g. mobile app, web app, etc...) and the `Supplier`s on Pocket Network, handling
+all intermediary business logic.
+
+A `Gateway` operator or a sovereign `Application` interested in accessing Pocket
```
Comment (red-0ne) [inline docusaurus/docs/actors/appgate_server.md:18]: What do you think about having `Gateway`, `Application` and `Supplier` mentions as links to their respective pages?

File: docusaurus/docs/actors/gateway.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+- [Overview](#overview)
+- [Schema](#schema)
+- [Configuration](#configuration)
+- [CLI](#cli)
+
+## Overview
+
+An `Gateway` is responsible for staking POKT in order to relay and sign
+requests on behalf of an `Application`.
```
Comment (red-0ne) [inline docusaurus/docs/actors/gateway.md:16]: What about making the word `Application` a link to `docs/actors/application.md`

File: docusaurus/docs/actors/supplier.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+## CLI
+
+All of the read (i.e. query) based operations for the `Supplier` actor can be
+viewed by running:
+
+```bash
+poktrolld query supplier
+```
+
+All of the write (i.e. tx) based operations for the `Application` actor can be
```
Comment (red-0ne) [inline docusaurus/docs/actors/supplier.md:35]:
```suggestion
All of the write (i.e. tx) based operations for the `Supplier` actor can be
```

File: docusaurus/docs/actors/gateway.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+## CLI
+
+All of the read (i.e. query) based operations for the `Gateway` actor can be
+viewed by running:
+
+```bash
+poktrolld query gateway --help
+```
+
+All of the write (i.e. tx) based operations for the `Application` actor can be
```
Comment (red-0ne) [inline docusaurus/docs/actors/gateway.md:35]:
```suggestion
All of the write (i.e. tx) based operations for the `Gateway` actor can be
```

**Review by red-0ne — CHANGES_REQUESTED** [review]

**Comment by red-0ne** [comment]
@Olshansk I addressed the change requests in 0591dcb7b755f517c2d8c2bb0550bec3ad01a252

PTAL and make modification if/where needed.

**Review by red-0ne — APPROVED** [review]

**Comment by Olshansk** [comment]
@red-0ne FYI: https://github.com/pokt-network/poktroll/commit/5a0981da72eed32763fd57fe8f40867f3d5e2a82

**Pull request merged**